specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2742 @@
+          },
+          {
+            "in": "body",
+            "name": "trainingJobCreateObject",
+            "description": "The training input parameters.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TrainingJobCreate"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "operation-location header for the status of the job.",
+            "headers": {
+              "operation-location": {
+                "description": "The location of the status API for monitoring the created job.",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "404": {
+            "description": "The project does not exist."
+          },
+          "409": {
+            "description": "Training job already inprogress."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    },
+    "/projects/{projectName}/deployments": {
+      "get": {
+        "description": "List all deployments.",
+        "operationId": "Deployments_GetDeploymentsList",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "projectName",
+            "description": "The project name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "top",
+            "description": "The number of deployments to be returned.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "The offset in the response.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxpagesize",
+            "description": "The maximum number of deployments per page.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of all deployments.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentObjectList"
```

Comment:
This API appear to be pagination. Please use appropriate extension https://azure.github.io/autorest/extensions/#x-ms-pageable